Neos.Flow/Tests/Unit/Property/TypeConverter/CollectionConverterTest.php

```diff
@@ -50,9 +50,10 @@ public function getTypeOfChildPropertyReturnsElementTypeFromTargetTypeIfGiven()
 
     /**
      * @test
+     * @expectedException \Neos\Flow\Property\Exception\InvalidDataTypeException
      */
     public function getTypeOfChildPropertyReturnsEmptyStringForElementTypeIfNotGivenInTargetType()
     {
-        $this->assertEquals('', $this->converter->getTypeOfChildProperty('array', '', $this->createMock(PropertyMappingConfigurationInterface::class)));
+        $this->converter->getTypeOfChildProperty('array', 'collection', $this->createMock(PropertyMappingConfigurationInterface::class));
```

Comment:
The test method name is not correct any more, and this changed expectation makes me wonder: Is just `Collection` not a valid annotation any longer?

Reply:
It is generally a valid annotation, but it's invalid in terms of property mapping, because without the element type the mapping can't continue into the elements (and will throw with an unspecific exception then). TBH I'm not sure why there was a test for this to return an empty string as the element type, other than assuming the test was just fixating the straightforward implementation.

Reply:
Of course, that makes sense!